cookies/operations.py

```diff
@@ -462,8 +428,19 @@ def generate_collection_coauthor_graph(collection,
 
     return graph
 
-@mock.patch('requests.get')
-def concept_search(text, mock_get):
+
+def ping_remote_resource(path):
```

Comment:
To be clear: the reason that we don't want the function defined twice is that it creates a maintenance headache. For example, if we later discover a bug in the function, or want to extend it, we have to do it in both places. Or risk having two different functions that two almost the same thing, with almost the same name. It can get nightmarish fast.